src/core/friendly_errors/fes_core.js

```diff
@@ -731,6 +731,38 @@ if (typeof IS_MINIFIED !== 'undefined') {
             );
             break;
           }
+          case 'REDECLAREDVARIABLE': {
+            let errSym = matchedError.match[1];
+            let url =
+              'https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Errors/Redeclared_parameter#what_went_wrong';
+            p5._friendlyError(
+              translator('fes.globalErrors.syntax.redeclaredVariable', {
+                symbol: errSym,
+                url
+              })
+            );
+            break;
+          }
+          case 'MISSINGINITIALIZER': {
+            let url =
+              'https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Errors/Missing_initializer_in_const#what_went_wrong';
+            p5._friendlyError(
+              translator('fes.globalErrors.syntax.missingInitializer', {
+                url
+              })
+            );
+            break;
+          }
+          case 'BADRETURNORYIELD': {
+            let url =
+              'https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Errors/Bad_return_or_yield#what_went_wrong';
+            p5._friendlyError(
+              translator('fes.globalErrors.syntax.badReturnOrYield', {
+                url
+              })
+            );
+            break;
+          }
```

Comment:
Please add inline comments explaining what each case is 👍

Reply:
I added the comments, please have a look

Reply:
awesome job!